### ISSUE TITLE
bit start errors on "Cannot find module 'postcss-loader'" - 0.0.801

### ISSUE THREAD

**Issue by jonatankBIT** [teambit/bit#6206]
### Describe the bug
A clear and concise description of the issue.

### Steps to Reproduce
1. bit new react-native x-sample
1. bit install && bit compile
1.bit start

### Expected Behavior
Start of the server, instead got error:

```
** unhandled rejection found, please make sure the promise is resolved/rejected correctly! **
Cannot find module 'postcss-loader'
Require stack:
- /Users/jkru/.bvm/versions/0.0.801/bit-0.0.801/node_modules/@teambit/react/dist/webpack/webpack.config.base.js
- /Users/jkru/.bvm/versions/0.0.801/bit-0.0.801/node_modules/@teambit/react/dist/react.env.js
- /Users/jkru/.bvm/versions/0.0.801/bit-0.0.801/node_modules/@teambit/react/dist/react.main.runtime.js
- /Users/jkru/.bvm/versions/0.0.801/bit-0.0.801/node_modules/@teambit/bit/dist/load-bit.js
- /Users/jkru/.bvm/versions/0.0.801/bit-0.0.801/node_modules/@teambit/bit/dist/app.js
- /Users/jkru/.bvm/versions/0.0.801/bit-0.0.801/node_modules/@teambit/bit/bin/bit
```

### Screenshots, exceptions and logs
![image](https://user-images.githubusercontent.com/97826324/181180574-7150335d-3f48-49ea-8cfa-624e122248e1.png)

### Specifications
* Bit version: 0.0.801
* Workspace type: (harmony/legacy) harmony
* Node version: 16.14.0
* npm / yarn version: 1.22.18
* Platform: MacOs

_for harmony workspace_

* relevant env : teambit.react/react-native
* relevant aspects

**Comment by benjgil** [comment]
@jonatankBIT 
This sort of error, about a random missing dependency that the user shouldn't need to install themselves, is almost always because the user has a core bit module in their dependencies list in workspace.jsonc or package.json - this will corrupt your bit installation, causing dependencies to not be found.

**tl;dr** 
Resolution:
Remove any packages of the form "@teambit/package" (i.e. ones which have only a single slash and no this.that, e.g. "@teambit/harmony") from your dependencies list, then manually delete your bit installation from the `.bvm/versions` folder in your root user directory.
Then re-install bit with `bvm upgrade`. 
Delete the node_modules in your workspace, re-install and compile and this should be resolved.

**The detail**
These core modules are already brought into your workspace via symlinks to the core bit installation. If you install the component manually, e.g. by running `bit install @teambit/harmony` for instance, the contents of this symlink become corrupted and the component and its dependencies are not where they should be (i.e. in the core bit folder on your machine).
This causes the bit components, which are only symlinked to your workspace, to not have the dependencies they need and they then report the kind of error you're seeing.
